src/instructions/einfuehrung_in_vat.md

```diff
@@ -72,53 +70,57 @@ First we go to the data selection menu and search for _Aeshna affinis_ in the GB
 
 <img src="images/einfuehrung_in_vat_13.png" width="50%" height="auto" alt="GBIF Search">
 
-To link the occurrence data with the mean temperature, we search for the _Minimum Temperature_ dataset in the data catalogue.
+**Warning:** Note that the GBIF data is spatio-temporal data. This means that a time period must be requested with an end date different from the start date.
```

Comment:
That's not entirely true. When start and end date are equal the range is treated as an instant which can return data. But it just returns data valid at exactly this moment and for the GBIF data this means that the event date must exactly match the instant, which makes it very unlikely to find data at some instant unless you already know that there are occurrences with this event date. So maybe mention that the occurrences are happening at a specific point in time and you are interested in all occurrences within a time range.